docs/Displaying a User Interface/Media Clock/index.md

```diff
@@ -3,6 +3,8 @@ The media clock is used by media apps to present the current timing information
 
 The media clock consists of three parts: the progress bar, a current position label and a remaining time label. In addition, you may want to [update the play/pause button icon](#setting-the-play-pause-button-style-rpc-v50) to reflect the current state of the audio [or the media forward / back buttons](#setting-the-media-forward-back-button-style-rpc-v71) to reflect if it will skip tracks or time.
 
+Media clock operations require the HMI status to be `FULL`. More information on how to monitor the HMI status can be found in the [Understanding Permissions](Getting Started/Understanding Permissions#monitoring-the-hmi-level) guide.
```

Comment:
This might be better put as a `!!! NOTE` . Also it might be better to have the like go to the `HMI Levels` section that is right above the `Monitoring the HMI Level` section